experimental/traffic-portal/server.ts

```diff
@@ -129,7 +138,6 @@ function run(): number {
 			" verification of any passed SSL keys/certificates"
 	});
 	parser.add_argument("-p", "--port", {
-		default: 4200,
```

Comment:
What's the new default? (Also need to change the help text below)

Reply:
It is still 4200. If this is present this port will [_always_ supersede](https://github.com/apache/trafficcontrol/blob/75c37693f80bc774ca175a8720d418d983bc4ea3/experimental/traffic-portal/server.config.ts#L326-L328) what is in the config file. I'd be happy to change the parsing instead of removing the argparser default if you'd like

Reply:
I'm just going to change the parsing code, this is not the only occurrence of this issue (defaults overriding config file values).

Reply:
Excellent, because I would like.